src/functions/displayInputGuide.js

```diff
@@ -0,0 +1,11 @@
+import MESSAGES from '../assets/message.js';
+
+const displayInputGuide = function displayInputGuideFunc() {
+  console.log(MESSAGES.GUIDE.HEADER);
+  console.log(MESSAGES.GUIDE.BODY);
+  console.log(MESSAGES.GUIDE.FOOTER);
+  console.log(MESSAGES.GUIDE.DELIMITER_INFORM);
+  console.log(MESSAGES.GUIDE.ALLOWED_SPECIAL_CHARACTERS);
```

Comment:
`console.log` 대신 우아한테크코스 유틸 라이브러리의 `Console` API를 활용하는 방법을 추천드립니다.
프로그래밍 요구사항 만족에도 적합하고 `print` 메서드로 `console.log`와 유사한 기능을 구현할 수 있습니다!

[라이브러리 링크]
https://github.com/woowacourse-projects/javascript-mission-utils?tab=readme-ov-file#printmessage

```suggestion
  Console.print(MESSAGES.GUIDE.HEADER);
  Console.print(MESSAGES.GUIDE.BODY);
  Console.print(MESSAGES.GUIDE.FOOTER);
  Console.print(MESSAGES.GUIDE.DELIMITER_INFORM);
   Console.print(MESSAGES.GUIDE.ALLOWED_SPECIAL_CHARACTERS);
```

Reply:
아 모듈을 주는 만큼 Console.print를 사용하는 것이 좋겠네요!! 좋은 의견 감사드립니다!! 👍